docs/spfx/use-aadhttpclient.md

```diff
@@ -188,12 +188,12 @@ Global and SharePoint administrators can also use the SharePoint Online Manageme
   > [!NOTE]
   > SharePoint doesn't verify if the requested permissions have already been granted or not, so before approving or rejecting a permission request, check which permissions have already been granted in your tenant.
 
-- To *approve the specific permission request*, use the **Approve-SPOTenantServicePrincipalPermissionRequest -RequestId <Guid>** cmdlet, specifying the ID of the permission request that you want to approve.
+- To *approve the specific permission request*, use the **Approve-SPOTenantServicePrincipalPermissionRequest -RequestId &lt;Guid&gt;** cmdlet, specifying the ID of the permission request that you want to approve.

```

Comment:
Correct markdown escaping of angled brackets should be prefixed with`\`; don't change to the HTML encoding.